Formula/lilv.rb

```diff
@@ -24,4 +24,7 @@ def install
     system "./waf"
     system "./waf", "install"
   end
+
+  test do
```

Comment:
A test would be good here, even something as simple as `--help` or `--version` (although more complex tests are preferred).